dice/dice.class.old.php

```diff
@@ -0,0 +1,340 @@
+//		if (!ereg("^[_a-zA-Z0-9-]+(\.[_a-zA-Z0-9-]+)*@[a-zA-Z0-9-]+\.([a-zA-Z0-9-]{2,3})$",$email)) {
+		$regex = "/^[_a-zA-Z0-9]+(\.[_a-zA-Z0-9-]+)*@[a-zA-Z0-9-]{2,}(\.[_a-zA-Z0-9-]+)?\.([a-zA-Z0-9-]{2,3})$/";
+			/*	legal examples: name@domain.com, my.name@domain.net, name@subdomain.domain.de, my.name@subdomain.domain.org
+			 *
+			 * 	/^[_a-zA-Z0-9-]+		begins with letter or number
+			 * 	(\.[_a-zA-Z0-9-]+)*		none or multiple letters or numbers which begins with a .
+			 * 	@						@
+			 * 	[a-zA-Z0-9-]{2,}		at least two more letters or numbers
+			 * 	(\.[_a-zA-Z0-9-]+)?		optional: at least one more char or num which begins with a .
+			 * 	\.						.
+			 * 	([a-zA-Z0-9-]{2,3})$/	2-3 chars or numbers and end of expression
+			 */
+		if (!preg_match($regex,$email)) {
+			echo "fatal error: email $email has wrong format!";
+			return false;
+		}
+
+		//echo "email $email is ok<br>";
+		return true;
+	}
+
+
+
+	function getDate() {
+		return date("Y-m");
+	}
+
+	function sendEmail($emails, $subject, $dice, $iv, $encrypted_data) {
+
+		$to  = implode (", ", $emails);
+		$date = $this->getDate();
+
+		// send email to member
+		$message = "Your dice are: $dice \n";
+		$message .= "Have a nice day. \n\n";
+		$message .= "----------------------------------\n";
+		$message .= "This is an automatically created email of the TripleA Ladder. Please don't reply to it. \n\n";
+		$message .= "Verification Info: Follow this link to check if your dice are authentic \n";
+		$message .= $this->domain."/MARTI_verify.php?date=$date&iv=$iv&enc=$encrypted_data \n\n";
+		$message .= "*** $date *** \n";
+		$message .= "$iv \n";
+		$message .= "............. \n";
+		$message .= "$encrypted_data \n";
+		$message .= "************* \n\n\n";
+ 		$message .= "----------------------------------\n";
+ 		$message .= "This Email is not SPAM.\nYou receive this email because of your registration at ".$this->domain."\n";
+		$message .= "To unsubscribe from this service go to ".$this->domain."/unsubscribe.php";
+
+		$ehead= "From: MARTI<marti@tripleawarclub.org>\r\n";
+		 $ehead .= "List-Unsubscribe:<http://dice.tripleawarclub.org/unsubscribe.php>\r\n";
+
+
+		$subj = "$subject";
+
+		$mailsend= @mail("$to","$subj","$message","$ehead");
+
+    /*$fd = popen("/usr/sbin/sendmail -t","w") or die("Couldn't Open Sendmail");
+    fputs($fd, "To: ".$to." \n");
+    fputs($fd, "From: \"MARTI\" <marti@tripleawarclub.org> \n");
+    fputs($fd, "Subject: ".$subject." \n");
+    fputs($fd, "X-Mailer: PHP3 \n\n");
+    fputs($fd, $message);
+    pclose($fd);*/
+
+		if ($mailsend) {
+				echo("<p>Dice results were sent via email!</p> <br> <a href=\"http://dice.tripleawarclub.org/MARTI_verify.php?date=$date&iv=$iv&enc=$encrypted_data\">click here to verify the roll</a><br>");
+			}
+		else {
+		   echo("<p>Email delivery failed...</p> Dice results were not sent. <br> Please try it later again.");
+		   exit("<p>fatal error: email delivery failed!");
+		}
+
+	}
+
+
+}
+?>
```

Comment:
The dice.class.old.php file used to be the main  dice.class.php file and the current one was the dice.class.new.php file. 
When I upgraded the server and PHP 7 was required, the old script no longer worked, but luckily there was already a new script compatible with PHP 7.
TL;DR This file is unused and broken, it can be removed in another commit